examples/MinioClientWithClientGrantsProvider.java

```diff
@@ -72,42 +70,20 @@ public static void main(String[] args) throws Exception {
         "https://IDP-HOST:IDP-PORT/auth/realms/master/protocol/openid-connect/token";
 
     // Client-ID to fetch JWT.
-    String clientId = "USER-ID";
+    String clientId = "CLIENT-ID";
 
     // Client secret to fetch JWT.
-    String clientSecret = "PASSWORD";
-
-    // Client-ID of MinIO service on IDP.
-    String idpClientId = "MINIO-CLIENT-ID";
+    String clientSecret = "CLIENT-SECRET";
```

Comment:
The client ID/secret are account information of IDP service. Either you leave it as is or change `User-ID` to `Username`

Reply:
Since they are passed as `client_id` and `client_secret` to IDP, I can name them as `IDP-CLIENT-ID` and `IDP-CLIENT-SECRET`. Calling it as USER-ID will create confusion.